.circleci/config.yml

```diff
@@ -100,11 +101,44 @@ jobs:
                 to: 's3://${AWS_BUCKET}/${CIRCLE_TAG-$CIRCLE_BRANCH}-$(echo $CIRCLE_SHA1 | cut -c -7)/index.html'
                 arguments: |
                   --cache-control max-age=0
+            # Copy to a "live" develop folder
+            - aws-s3/copy:
+                from: dist
+                to: 's3://${AWS_BUCKET}/develop/'
+                arguments: |
+                  --recursive \
+                  --exclude index.html
+            - aws-s3/copy:
+                from: dist/index.html
+                to: 's3://${AWS_BUCKET}/develop/'
+                arguments: |
+                  --cache-control max-age=0
+  update_cloudfront:
+    working_directory: ~/repo
+    docker:
+      - image: circleci/python:2.7
```

Comment:
hmmm, rats
https://github.com/aws/aws-cli/issues/3553

Reply:
There's a [Circle CI orb](https://circleci.com/orbs/registry/orb/circleci/aws-cli) for AWS-CLI, but it has no examples of how to invoke a command. So I gave up and went this route. :(